### PR TITLE
KAA-1571: Extend documentation for Edison platform according to QA Review

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @GeorgePostolakiy @GamovCoder @yverzun

### CODE REVIEW

**Comment by yverzun** [comment]
LGTM

**Review by yverzun — APPROVED** [review]
LGTM

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs
 
         tar -xvf edison-toolchain-20150120-linux64.tar.bz2
 
-2. Install toolchain.
+1. Install toolchain.
```
Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: `s/toolchain/the toolchain/`

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md [hunk 2]
```diff
@@ -36,17 +36,23 @@ Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs
 
         sed -i 's:+111:/111:' install_script.sh
 
-    The cross compilation toolchain is installed to `/opt/poky-edison/1.6.1/` directory by default. On some configurations the script installs the toolchain only to its working directory.
+    The cross compilation toolchain is installed to current directory by default. On some configurations the script installs the toolchain to `/opt/poky-edison/1.6.1/`.
```
Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: `s/current directory/the current directory`

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md [hunk 3]
```diff
@@ -36,17 +36,23 @@ Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs
...
+
+1. Install [cmake](https://cmake.org/):
```
Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: `s/cmake/CMake/`

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md [hunk 4]
```diff
@@ -36,17 +36,23 @@ Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs
...
 Now, dependencies are installed and it is time to create Kaa application.
 Since Edison is running Linux, you can refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/#c-sdk-build) for detailed process of application creation.
 But remember, you must specify correct compiler when compiling your Kaa application for Intel Edison:
 
-        cmake -DKAA_MAX_LOG_LEVEL=3 -DCMAKE_TOOLCHAIN_FILE=PATH_TO_KAA_SDK/toolchains/edison.cmake ..
-        make 
+        cmake -DKAA_MAX_LOG_LEVEL=3 -DCMAKE_TOOLCHAIN_FILE=PATH_TO_KAA_SDK/toolchains/edison.cmake -DEDISON_SDK_ROOT=PATH_TO_EDISON_SDK -DBUILD_TESTING=OFF ..
+        make
+
+>**NOTE:** `PATH_TO_KAA_SDK` should be replaced with path to Kaa C SDK relative to the `build` directory
```
Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: `s/with path to Kaa C SDK/with the path to Kaa C SDK`
`s/relative/relatively/`

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md [hunk 5]
```diff
@@ -36,17 +36,23 @@ Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs
...
+and `PATH_TO_EDISON_SDK` should be replaced with absolute path to the Edison SDK installation directory (see above).
```
Comment (ohamov) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Edison/index.md]: `s/with absolute path/with the absolute path/`

**Review by ohamov — CHANGES_REQUESTED** [review]

**Review by ohamov — APPROVED** [review]

**Pull request merged**